### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Bovard Nicolas - Froidevaux Kilian

### DISCUSSION THREAD

**Discussion by kilianfroideva** [heig-vd-dai-course/heig-vd-dai-course#554] [Show and tell]
We will implement a Sudoku web application where we can do the following:      
- Play at sudoku 9x9 with different difficulty or 16x16
- Multiplayer sudoku race where the fastest to complete the grid wins

APIs:
- Create a new Sudoku game
- Join an existing game
- Update player progress
- Retrieve game state
- End the game and determine the winner

To go further:
- Timer
- Leaderboard

**Comment by GeraudSilvestri** [comment]
Good for me

**Comment by kilianfroideva** [comment]
Repo: https://github.com/kilianfroideva/heig-vd-dai-course-java-practical-content-3-sudoku
Commit: 19bbcb4abc85600985965cdb36f6417f7b08431a

**Reply by ludelafo** [reply]
Bonjour @kilianfroideva,

Je n'ai pas accès au repo. S'il est privé, merci de nous y inviter avec Géraud et Hadrien.

Merci et à tout de suite !
